### DISCUSSION TITLE
index_i and index_j in message passing layer flipped in the new version

### DISCUSSION THREAD

**Discussion by abmukho** [danielegrattarola/spektral#239] [General]
While comparing my old trained model using crystal conv after implementing the new changes from the new release, I noticed that the index_i and index_j are flipped in the new version in the message_passing code. For example in the previous release, index_i was a.indices[:, 0] which intuitively made more sense because with x_i~get_i (self) and x_j~get_j(neighbours), the message passing was performed using convolution done on x_i||x_j||e_ij which you then pool on unique x_i's using deserialize scatter. Is there anything subtle in the new release that I am missing? Few explanations that I came up with are that either e_ij is now actually e_ji or a_ij is now a_ji... which I didn't see in the code. What am I missing?

**Comment by danielegrattarola** [comment]
Duplicate of #249 
I've replied there
